bindings/wasm/tests/legacy/credentials.ts

```diff
@@ -1,3 +1,4 @@
+// REFACTOR-TODO: Reactivate after credential stuff is fixed.
```

Comment:
Please make an issue or something to track all these TODOs.

Reply:
All of the `REFACTOR-TODO`s are intended to be resolved before merging this PR. Most of them are about the credential module in Wasm that needs to be updated to no longer use the legacy `Iota` types but the new ones. It was supposed to be fixed by now, but there was a misunderstanding apparently. I might end up converting these to `TODO`s and keep track of them in an issue or somewhere else.